<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatRecordDialog</class>
 <widget class="QDialog" name="ChatRecordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>440</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>0</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="windowTitle">
   <string>ChatRecordDialog</string>
  </property>
  <widget class="QWidget" name="widgetMenu" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>430</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>430</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>25</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.468, y1:0, x2:0.473, y2:1, stop:0 rgba(221, 255, 255, 255), stop:1 rgba(240, 255, 253, 255));</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>聊天记录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClose">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>10</width>
      <height>10</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>10</horstretch>
      <verstretch>10</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border-image: url(:/RecordClose.png);
}
QPushButton:hover
{
	border-image: url(:/RecordCloseHover.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMore">
    <property name="geometry">
     <rect>
      <x>327</x>
      <y>0</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>更多</string>
    </property>
    <property name="icon">
     <iconset resource="images/images.qrc">
      <normaloff>:/more.png</normaloff>:/more.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widgetTool" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>430</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>430</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>31</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.477, y1:1, x2:0.468, y2:0, stop:0 rgba(221, 255, 255, 255), stop:1 rgba(240, 255, 253, 255));

</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>查找</string>
    </property>
    <property name="icon">
     <iconset resource="images/images.qrc">
      <normaloff>:/find.PNG</normaloff>:/find.PNG</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFirstPage">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>首</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLeftPage">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>左</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLastPage">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>尾</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRightPage">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>右</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelDate">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>20XX-XX-XX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDate">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/images.qrc">
      <normaloff>:/birthday.png</normaloff>:/birthday.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>页/第      页</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPage">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>420</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>第</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPage">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>410</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>999</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>431</width>
     <height>390</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>431</width>
     <height>390</height>
    </size>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>textEdit</zorder>
  <zorder>widgetTool</zorder>
  <zorder>widgetMenu</zorder>
  <zorder>label_2</zorder>
  <zorder>lineEditPage</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images/images.qrc"/>
 </resources>
 <connections/>
</ui>
